streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -406,6 +406,20 @@ void writeAsText(final String filePath,
     void foreach(final ForeachAction<? super K, ? super V> action);
 
     /**
+     * Perform an action on each record of {@code KStream}.
+     * This is a stateless record-by-record operation (cf. {@link #process(ProcessorSupplier, String...)}).
+     *
```

Comment:
Nit: add `<p>` to get an empty line -- otherwise, generated HTML does not render correctly.
Same below.